modules/manage/pages/manage-buckets/migrate-bucket.adoc

```diff
@@ -185,3 +192,122 @@ include::manage:example$migrate-bucket-storage-backend.sh[tag=rebalance-cluster]
 ----
 
 . Repeat the previous step until all nodes that you'd migrated have rolled back to their original storage backend. 
+
+
+[#xdcr-migration]
+== XDCR Storage Backend Migration
```

Comment:
This section looks good to me but would like Neil Huang, the XDCR eng team manager to take a look as well.
cc @nelio2k

Reply:
LGTM